charts/studio/values.yaml

```
@@ -76,6 +76,12 @@ global:
     databasePassword: "postgres"
 
   scmProviders:
+
+    # -- Set custom hostname for incoming webhook  if you're using SaaS version of GitHub, GitLab or Bitbucket.
```

```suggestion
    # -- Custom hostname for incoming webhook (if Studio runs on a private network and you use SaaS versions of GitHub, GitLab, or Bitbucket)
```

We need to reword the description because it's technically not a requirement if they deployed Studio on a public domain (this is probably the minority of users, but it would be confusing to them).